NuGet.config

```diff
@@ -41,6 +43,7 @@
   <packageSourceMapping>
     <clear />
     <packageSource key="general-testing">
+      <package pattern="microsoft.*" />
```

Comment:
Is it possible to narrow this down a bit more? Does `microsoft.codeanalysis.*` work?

Reply:
I think that would work. Note that I will be able to revert this change altogether once https://github.com/dotnet/roslyn/pull/66570 gets merged & published. Then I could just bump Roslyn version (previously, I couldn't do that because updating to Roslyn 4.6 broke CI, but not it seems to work - see #8244).

Reply:
I don't think there is much security risk with this. In fact, more realistically, we shouldnt have `general-testing` as a source in nuget.config at all. AFAIK thats used for testing packages from PR builds, and we shouldnt do that

Reply:
The source actually wasn't here, I added it in this PR, just the package mapping was there (probably a leftover). I can also of course just push the package to another source, and then remove this.

Reply:
Definitely a leftover then, probably from before CLaSP was in a real feed. I think it would be best to move the package to a real feed, and remove `general-testing` source and mapping, but i'm sure that will take some negotiation so doesn't have to be in this PR.

Reply:
Since https://github.com/dotnet/roslyn/pull/66570 has been merged, I'm able to use official versions, so I removed this.